### ISSUE TITLE
Do not warn about OpenShift resources as not being a k8s resource

### ISSUE THREAD

**Issue by zivnevo** [np-guard/cluster-topology-analyzer#204]
Currently the warnings look like
```shell
2023/05/04 17:03:05 in file: frontend\asset-cache\route.yaml, document: 0, Yaml document is not a K8s resource: no kind "Route" is registered for version "route.openshift.io/v1" in scheme "pkg/runtime/scheme.go:100"
```
